pages/docs/components/typography/heading.mdx

```diff
@@ -67,8 +67,9 @@ the default theme. This can be seen in the [theme source](https://github.com/cha
 
 ### Truncate heading
 
-Pass the `noOfLines={1}` prop to render an ellipsis when the heading exceeds the
-width of the viewport or `maxWidth` prop.
+If you'd like to truncate the heading after a specific number of lines, pass the
+`noOfLines` prop. This will render an ellipsis when the heading exceeds the width
+of the viewport or `maxWidth` prop.
```

Comment:
Also updated Heading docs to be more clear that `noOfLines` accepts any number, not just `noOfLines={1}`